### ISSUE TITLE
RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by zee-shan000** [nuxt/nuxt#26605]
### Environment

Dev

### Reproduction

No reproduction.

### Describe the bug

Getting the below error after updating my nuxt version from ('^3.5.1') to ('^3.11.1).
and it will break the page and show "This page isn't working", after reloding it work's fine.

Error is here.
```
browser.mjs?v=fd19e36f:44 Uncaught (in promise) RangeError: Maximum call stack size exceeded
    at BrowserReporter.log (browser.mjs?v=fd19e36f:44:7)
    at Consola._log (core.mjs?v=fd19e36f:381:16)
    at resolveLog (core.mjs?v=fd19e36f:349:14)
    at Consola._logFn (core.mjs?v=fd19e36f:377:5)
    at core.mjs?v=fd19e36f:306:19
    at BrowserReporter.log (browser.mjs?v=fd19e36f:44:7)
    at Consola._log (core.mjs?v=fd19e36f:381:16)
    at resolveLog (core.mjs?v=fd19e36f:349:14)
    at Consola._logFn (core.mjs?v=fd19e36f:377:5)
```

**Comment by ausir0726** [comment]
> Not much to do here without a reproduction. If it only happens with i18n, consider raising an issue in https://github.com/nuxt-community/i18n-module too (but also **with a reproduction**).

My situation currently only happened in i18n.
I think I should react over there.
I just happened to see the OP experiencing the same issue.
Thank you.

**Comment by ausir0726** [comment]
@manniL 

I reproduced this issue on codesandbox
https://codesandbox.io/p/devbox/fancy-wave-7tjy7r

Although it's currently only happening in i18n
Actually, my colleagues have similar issues in other situations, but they didn't record the reasons and have forgotten.

I have already reported and submitted it on the i18n side
But because this issue arises after 3.11.0
Everything was working fine before 3.10.3

After 3.11.0, I noticed
`WARN Cannot stringify arbitrary non-POJOs ComputedRefImpl`
And the screen no longer renders, only the NUXT LOGO appears, everything freezes

After 3.11.1, it throws Maximum call stack size exceeded

So I'm wondering if 3.11 might have broken something?

Thanks

**Comment by zee-shan000** [comment]
@DamianGlowala Yes I am using i18n module. 
`"@nuxtjs/i18n": "^8.1.1"`

**Comment by pothuraju-dev** [comment]
> @manniL
> 
> I reproduced this issue on codesandbox https://codesandbox.io/p/devbox/fancy-wave-7tjy7r
> 
> Although it's currently only happening in i18n Actually, my colleagues have similar issues in other situations, but they didn't record the reasons and have forgotten.
> 
> I have already reported and submitted it on the i18n side But because this issue arises after 3.11.0 Everything was working fine before 3.10.3
> 
> After 3.11.0, I noticed `WARN Cannot stringify arbitrary non-POJOs ComputedRefImpl` And the screen no longer renders, only the NUXT LOGO appears, everything freezes
> 
> After 3.11.1, it throws Maximum call stack size exceeded
> 
> So I'm wondering if 3.11 might have broken something?
> 
> Thanks

I am also facing same issue when using `@pinia/nuxt`. I created a project in stackblitz https://stackblitz.com/edit/nuxt-starter-kzk5zi but the issue it not reproducing in stackblitz. 

When I run in local system, I am getting the issue. My system configuration
```
OS: Ubuntu 20.04.6 LTS
Node: v18.18.2
npx: 9.8.1
```

I created project using command: `npx nuxi@latest init` and then installed `@pinia/nuxt` and `pinia`. I also added `@pinia-plugin-persistedstate/nuxt` package for persistance.

**Comment by ausir0726** [comment]
https://github.com/nuxt-modules/i18n/issues/2892#issuecomment-2033581580

The error was due to the new and improved logging in version 3.11.
It seems that this feature has overridden the behavior of console.log.
To restore normal function, simply disable devlogs.

I confirmed with my colleagues that there is also a chance of it happening other than i18n. [cannot reproduce]

```
export default defineNuxtConfig({
  features: {
    devLogs: false
    // or 'silent' to allow you to handle yourself with `dev:ssr-logs` hook
  },
})
```
